code/modules/mob/living/brain/brain_item.dm

```diff
@@ -161,6 +161,8 @@
 		O.attack(brainmob, user) //Oh noooeeeee
 
 	if(O.force != 0 && !(O.item_flags & NOBLUDGEON))
+		user.do_attack_animation(src)
+		playsound(loc, 'sound/effects/meatslap', 50)
```

Comment:
...Does this work? This doesn't have .ogg

Reply:
Local branch has it must have fucked something up.